commit 582d5734cfcd11de7563366f9788ea72803277e6
Author: Stefan Schantl <stefan.schantl@ipfire.org>
Date:   Sat Apr 4 15:33:58 2015 +0200

    perl.macro: Add "perl_version".
    
    This variable is used to store the current used perl version, which
    will be used in various external perl modules to track API dependencies.

diff --git a/macros/perl.macro b/macros/perl.macro
index b29e028..aa8fa81 100644
--- a/macros/perl.macro
+++ b/macros/perl.macro
@@ -7,6 +7,7 @@ perl_vendorarch = %(eval "$(%{perl} -V:installvendorarch)"; echo ${installvendor
 perl_vendorlib  = %(eval "$(%{perl} -V:installvendorlib)";  echo ${installvendorlib})
 perl_archlib    = %(eval "$(%{perl} -V:installarchlib)";    echo ${installarchlib})
 perl_privlib    = %(eval "$(%{perl} -V:installprivlib)";    echo ${installprivlib})
+perl_version    = %(eval "$(%{perl} -V:version)";           echo ${version})
 
 perl_all_paths  = {%{perl_vendorarch},%{perl_vendorlib},%{perl_archlib},%{perl_privlib}}
 
